easybuild/easyblocks/cp2kcray.py

```diff
@@ -1,806 +0,0 @@
-##
```

Comment:
Is this relevant for TSA?

Reply:
No, but we never used it anyway, so it's a good occasion to get rid of it